conf/etc/nginx/sites-available/default

```diff
@@ -17,16 +17,22 @@
 # Please see /usr/share/doc/nginx-doc/examples/ for more detailed examples.
 ##
 
+# Redirect to HTTPS
 server {
 	listen 80 default_server;
-	listen [::]:80 default_server ipv6only=on;
+	listen [::]:80 default_server;
+	server_name localhost;
+	return 301 https://$server_name$request_uri;
+}
+# HTTPS server
+#
+server {
+	listen 443;
+	server_name localhost ipv6only=on;
```

Comment:
A couple questions on this line:

* Is this webserver only listening on the loopback interface (`localhost`), and not on any external interfaces?
* Can `ipv6only` be set on the `server_name` directive, or should it be on the `listen` directive?
* Do we want `ipv6only` set to `on`? I may not be understanding [the flag](http://nginx.org/en/docs/http/ngx_http_core_module.html#listen) correctly, but it sort of reads like it might be setting the socket on 443 to *only* accept IPv6 connections.